### ISSUE TITLE
Bd needs a "related posts" feature

### ISSUE THREAD

**Issue by sutibun** [backdrop/backdrop-issues#1317]
Nearly all sites I come across the web has a "Related posts" feature at the end of an article. It usually has two components

- post title
- post thumbnail

We should either build this into Bd or create a module.

Besides other people, I want it for my site too. I'd be satisfied with the [Similar Entries](https://www.drupal.org/project/similar) module ported over. ^^)

**Comment by biolithic** [comment]
I often just make a block from a View using relationships with common
taxonomy terms between the posts.

Or, are you thinking of a (paying) solution like Disqus or Outbrain?  These
might be satisfy?  Or not?

On Thu, Oct 29, 2015 at 9:59 PM, Steven Villaverde <notifications@github.com

**Comment by sutibun** [comment]
> Or, are you thinking of a (paying) solution like Disqus or Outbrain?

No, not too familiar with them. I thought Disqus was just a commenting system?

> I often just make a block from a View using relationships with common taxonomy terms between the posts.

2 slight problems

1. Monkey doesn't really know how to use relationships in Views (barely got it working for something else I was trying to build and that was a long time ago)
2. But more importantly, it'd be better for users to turn on this feature (whether a core or contrib module) In Wordpress, I just enable a plugin, it does the heavily lifting and I just play with the settings. In Drupal, the closest I found that just "works" is the "Similar Entries" module. (Plus, I also like how it compares the content's text which takes some pressure from me being perfect in managing the taxonomy)
